example/package.json

```
@@ -21,7 +21,7 @@
     "react-native-keyboard-aware-scroll-view": "github:APSL/react-native-keyboard-aware-scroll-view#v0.9.2",
     "react-native-modal-datetime-picker": "^8.9.3",
     "react-native-webview": "^10.3.3",
-    "surveyjs-react-native": "git+https://bitbucket.org/shiftsmart/surveyjs-react-native.git"
+    "surveyjs-react-native": "git+https://github.com/shiftsmartinc/surveyjs-react-native"
```

same here can we use 
```suggestion
    "surveyjs-react-native": "github:shiftsmartinc/surveyjs-react-native"
```